### ISSUE TITLE
About the effect of meta-train

### ISSUE THREAD

**Issue by pyj8711** [icoz69/DeepEMD#34]
Hi, this is very good work.
After running the code, I find a problem of the meat-train.
I use the CUB dataset. With the number of epochs increasing during pre-train, the validation accuracy on 5-way 5-shot task also increases. This is normal.
However, during the meta-train, where the feature extractor also gets updated, I can't find accuracy improvement or meta-train loss decreasing. The test accuracy after meta-train is similar to the validation accuracy after pre-train. Is that normal?
So, my question is, does the meta-train take effect?
Thanks.

**Comment by pyj8711** [comment]
PS:  I used the opencv to calculate EMD.

**Comment by icoz69** [comment]
you may try a smaller lr at meta-training stage
